habitat-hitl/habitat_hitl/core/hitl_main.py

```diff
@@ -117,21 +117,28 @@ def hitl_headed_main(hitl_config, app_config, create_app_state_lambda):
         text_drawer_kwargs=text_drawer_kwargs,
     )
 
-    update_config(
-        app_config,
-        show_debug_third_person=show_debug_third_person,
-        debug_third_person_width=debug_third_person_width,
-        debug_third_person_height=debug_third_person_height,
-    )
+    # TODO: Move to LabDriver
```

Comment:
might as well do this todo before merging

Reply:
I will also refactor main viewport handling in a later pass.